test/runScript-mock-pMap.spec.js

```diff
@@ -12,21 +12,27 @@ const packageJSON = {
 }
 
 describe('runScript', () => {
+  afterEach(() => {
+    pMapMock.mockClear()
+  })
+
   it('should respect concurrency', () => {
+    pMapMock.mockImplementation(() => Promise.resolve(true))
+
     const res = runScript(['test'], ['test1.js', 'test2.js'], packageJSON, {
       chunkSize: 1,
       subTaskConcurrency: 1
     })
     res[0].task()
-    expect(pMap.mock.calls.length).toEqual(1)
-    const pMapArgs = pMap.mock.calls[0]
+    expect(pMapMock.mock.calls.length).toEqual(1)
+    const pMapArgs = pMapMock.mock.calls[0]
     expect(pMapArgs[0]).toEqual([['test1.js'], ['test2.js']])
     expect(pMapArgs[1]).toBeInstanceOf(Function)
     expect(pMapArgs[2]).toEqual({ concurrency: 1 })
   })
 
   it('should handle unexpected error', async () => {
-    pMap.mockImplementation(() => Promise.reject(new Error('Unexpected Error')))
+    pMapMock.mockImplementation(() => Promise.reject(new Error('Unexpected Error')))
 
     const res = runScript(['test'], ['test.js'], packageJSON)
     try {
```

Comment:
Let's use http://facebook.github.io/jest/docs/en/expect.html#tothrowerrormatchingsnapshot here?

Reply:
I'd prefer doing stylistic changes in a different PR.